toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -69,7 +81,13 @@ func canUseCacheForNode(pkgGraph *pkggraph.PkgGraph, node *pkggraph.PkgNode, pac
 
 	// Check if the node corresponds to an entry in packagesToRebuild
 	specName := node.SpecName()
-	canUseCache = !sliceutils.Contains(packagesToRebuild, specName, sliceutils.StringMatch)
+	// Some nodes have no VersionedPkg, but get the name if it exists otherwise use ""
```

Comment:
@PawelWMS Will need to check if this overlaps with your *_LIST update. May not be needed anymore.